docs/developers-notes.md

```diff
@@ -65,19 +65,22 @@ We will set the environment variables using kustomize rules in the file
 patches:
 - patch: |-
     - op: add
-      path: /spec/template/spec/containers/0/env
+      path: /spec/template/spec/containers/0/env/-
       value:
-      - name: "SAMBA_OP_SMBD_CONTAINER_IMAGE"
```

Comment:
Is this removal of '-' intentional?

Reply:
Never mind, I found the answer in the link you provided below. It looks good to me.